test/integration/AdminSpec.scala

```diff
@@ -126,10 +179,103 @@ class AdminSpec
 
   }
 
-  private case class TestPartConfig(partId: String,
-                                    url: String,
-                                    connectionPoolSize: Int,
-                                    commandKey: String,
-                                    commandKeyGroup: String,
-                                    proxy: String)
+  describe("Cache groups administration") {
+
+    describe("the listing page") {
+
+      it("should work fine") {
+        goTo(CacheGroupListPage)
+        pageTitle should include("Cache groups")
```

Comment:
Good idea. Added a check to make sure that groups that already exist are on the listing page.